bindings/wasm/src/account/crypto.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use wasm_bindgen::prelude::*;
+
+use identity::account::derive_encryption_key as derive_encryption_key_;
+use identity::account::EncryptionKey as EncryptionKey_;
+
+#[wasm_bindgen(js_name = EncryptionKey, inspectable)]
+pub struct WasmEncryptionKey(EncryptionKey_);
+
+#[wasm_bindgen(js_class = EncryptionKey)]
+impl WasmEncryptionKey {
+  #[wasm_bindgen(js_name = deriveEncryptionKey)]
+  pub fn derive_encryption_key(password: &str) -> WasmEncryptionKey {
+    WasmEncryptionKey(derive_encryption_key_(password))
+  }
+}
```

Comment:
The internal type of `EncryptionKey` is simply `[u8; 32]`. I think we should consider just using `Uint8Array` or add a converter function? How would a Javascript `Storage` implementation actually use this as a password, can they access the bytes through `inspectable`?